tests/test_span.py

```diff
@@ -140,15 +140,15 @@ def test_traceback_with_error():
     assert s.error
     assert 'by zero' in s.get_tag(errors.ERROR_MSG)
     assert "ZeroDivisionError" in s.get_tag(errors.ERROR_TYPE)
-    assert s.get_tag(errors.ERROR_STACK)
+    ok_("ZeroDivisionError: integer division or modulo by zero" in s.get_tag(errors.ERROR_STACK))
 
 def test_traceback_without_error():
     s = Span(None, "test.span")
     s.set_traceback()
     assert not s.error
     assert not s.get_tag(errors.ERROR_MSG)
     assert not s.get_tag(errors.ERROR_TYPE)
-    assert not s.get_tag(errors.ERROR_STACK)
+    ok_("tb = ''.join(traceback.format_stack(limit=limit))" in s.get_tag(errors.ERROR_STACK))
```

Comment:
thinking about that, maybe is it better to discard the last occurrences of the stack that includes our code? It's just because I think it's not so interesting to users that when they `set_traceback()`, part of our code is available in the traceback.

If you check `traceback.format_stack(limit=limit)`, it returns a Python list and you can use a view instead of the full list (eg: `traceback.format_stack(limit=limit)[:-2]` if we should discard the last 2 elements).

In that way our `dd-trace-py` code will not be in the traceback list. If you think it's fine, remember to increase the `limit` value by the same amount you're removing from the full list so if you're `[:-2]`, then `limit += 2`. Otherwise you ask for a limit of 20 but only 18 results are returned.